### PR TITLE
move serialization docs

### PR DESCRIPTION
**Pull request by mairaw**
Fixes Part 1 of #2770

### CODE REVIEW

File: xml/System.Xml.Serialization/XmlIgnoreAttribute.xml [hunk 1]
```diff
@@ -34,11 +34,11 @@
...
   
  For more information about using attributes, see [Attributes](http://msdn.microsoft.com/library/30386922-1e00-4602-9ebf-526b271a8b87).
```
Comment (rpetrusha) [inline xml/System.Xml.Serialization/XmlIgnoreAttribute.xml]: Change link to `~/docs/standard/attributes/`?

File: xml/System.Xml.Serialization/XmlTypeAttribute.xml [hunk 2]
```diff
@@ -31,14 +31,14 @@
...
   
 > [!NOTE]
->  If you set the <xref:System.Xml.Serialization.XmlTypeAttribute.IncludeInSchema%2A> property to `false`, the [XML Schema Definition Tool (Xsd.exe)](~/docs/framework/serialization/xml-schema-definition-tool-xsd-exe.md) does not include the type in the schema. By default, for each public class, the XSD tool generates a `complexType` and an element of that type.  
+>  If you set the <xref:System.Xml.Serialization.XmlTypeAttribute.IncludeInSchema%2A> property to `false`, the [XML Schema Definition Tool (Xsd.exe)](~/docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md) does not include the type in the schema. By default, for each public class, the XSD tool generates a `complexType` and an element of that type.  
   
  For more information about using attributes, see [Attributes](http://msdn.microsoft.com/library/30386922-1e00-4602-9ebf-526b271a8b87).
```
Comment (rpetrusha) [inline xml/System.Xml.Serialization/XmlTypeAttribute.xml]: Change link to `~/docs/standard/attributes/`?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@mairaw, this looks good. There is one formatting issue, a number of MSDN URLs that could be converted to xrefs, and a number of other MSDN URLs that could be converted to docs.

**Comment by mairaw** [comment]
@rpetrusha good catch on those links! Even though the comments were not really related to moving the docs, I made the fixes. Please review. I've done some global fixes, so the number of files impacted is probably bigger than the number of files you've given feedback to.

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @mairaw. This looks really good. It's ready to merge when you want to.

**Review by rpetrusha — APPROVED** [review]

**Comment by mairaw** [comment]
I've noticed I've missed one of the serializer msdn links (my VS Code was super slow yesterday). Fixed that one and I'll merge if builds looks good. Thanks @rpetrusha!

**Pull request merged**